beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -239,12 +243,15 @@ func (s *Service) sendLightClientOptimisticUpdate(ctx context.Context, signed in
 	if err != nil {
 		return 0, errors.Wrap(err, "could not create light client update")
 	}
+	optimisticUpdate, err := structs.LightClientOptimisticUpdateFromConsensus(update)
+	if err != nil {
+		return 0, errors.Wrap(err, "could not convert light client update")
+	}
 
 	// Return the result
-	result := &ethpbv2.LightClientOptimisticUpdateWithVersion{
-		Version: ethpbv2.Version(signed.Version()),
-		// TODO: Get back to this when revisiting events
-		//Data:    update,
+	result := &structs.LightClientOptimisticUpdateResponse{
+		Version: hexutil.EncodeUint64(uint64(signed.Version())),
+		Data:    optimisticUpdate,
```

Comment:
You should just send the `optimisticUpdate` to the event feed. This is because:
- the update has a `Version()` method so sending a version field is redundant
- using a struct from the `rpc` package is wrong because there can be multiple event listeners, not necessarily from `rpc`